lexxpluss_apps/src/board_controller.cpp

```diff
@@ -1717,12 +1741,12 @@ class state_controller { // Variables Implemented
             gpio_pin_set_dt(&gpio_dev, bat_out_state);
             ac.set_enable(false);
             charge_guard_asserted = true;
+            use_software_brake = false;
             k_timer_start(&charge_guard_timeout, K_MSEC(10000), K_NO_WAIT); // charge_guard_asserted = false after 10sec
         } break;
         case POWER_STATE::SUSPEND: {
             LOG_INF("enter SUSPEND\n");
             psw.set_led(true);
-            wsw.set_disable(true);
```

Comment:
Proposals for the future
- (Sorry if this already exists.) Create a state transition diagram.
- Divide the SUSPEND state into two. Rather than updating use_software_brake before transitioning to the SUSPEND state, I think it would improve maintainability to set use_software_brake when entering the SUSPEND state.

今後に向けた提案
- (すでに存在していたらすみません。) 状態遷移図を作成する。
- SUSPEND状態を2つに分ける。SUSPEND状態への遷移前にuse_software_brakeを更新するより、SUSPEND状態のenterのタイミングでuse_software_brakeを設定すると保守性が向上するように思います。

Reply:
> SUSPEND状態を2つに分ける。SUSPEND状態への遷移前にuse_software_brakeを更新するより、SUSPEND状態のenterのタイミングでuse_software_brakeを設定すると保守性が向上するように思います。

この場合、SUSPENDになった理由の取得が困難になるためこのような実装としました。（software_brakeを使わずにSUSPEND状態になるケースとの区別のため）